src/net/sparkzz/modest/io/config/YAMLConfig.java

```diff
@@ -18,7 +18,6 @@
 
 	private DumperOptions dumperOptions;
 	private File configLocation;
-	private int indent = 2;
```

Comment:
Always good practice to avoid class-wide variables whenever possible. Here, indent is only used in one method, so its irrelevant to the rest of the class.